src/Coupon/Contracts/CouponHandler.php

```diff
@@ -5,6 +5,7 @@
 use Laravel\Cashier\Coupon\Coupon;
 use Laravel\Cashier\Coupon\RedeemedCoupon;
 use Laravel\Cashier\Exceptions\CouponException;
+use Laravel\Cashier\FirstPayment\Actions\ActionCollection;
```

Comment:
This use statement isn't used anywhere as far as I can see

Reply:
Thanks, I'm cleaning up everything now before release here: #73 .